cypress/support/commands.ts

```diff
@@ -112,7 +112,6 @@ Cypress.Commands.add('drag', (selector, pos, options) => {
 			x: options?.abs ? xDiff : originalLocation.right + xDiff,
 			y: options?.abs ? yDiff : originalLocation.top + yDiff,
 		}
-
 		if(options?.realMouse) {
```

Comment:
Would it also work if you move out the click from the `else` block? Just in case we would need that `mouseup` anyways and maybe an additional click on the body wouldn't hurt

Basically just keep the original event handling and if `clickToFinish` is passed you just make that additional click.
Like this:
```suggestion
		if(options?.realMouse) {
			element.realMouseDown();
			element.realMouseMove(newPosition.x, newPosition.y);
			element.realMouseUp();
		} else {
			element.trigger('mousedown', {force: true});
			element.trigger('mousemove', {
				which: 1,
				pageX: newPosition.x,
				pageY: newPosition.y,
				force: true,
			});
			lement.trigger('mouseup', {force: true});
		}

		if (options?.clickToFinish) {
			// Click to finish the drag
			// For some reason, mouseup isn't working when moving nodes
			cy.get('body').click(newPosition.x, newPosition.y);
		}
```

Reply:
Nope, for some reason, that `mouseup` is not ending the drag properly